### PR TITLE
Homework14: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**
homework_14_Files_Exceptions_Modules

### CODE REVIEW

File: hw_14/1.py
Comment (Stml89) [inline hw_14/1.py:1]: not really good module name

File: hw_14/2.py
Comment (Stml89) [inline hw_14/2.py:1]: Good, but it does not take operator's priority into account:
1/ 2 + 2 * 2
2/ 2 ** 3 - 1

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hw_14/1.py
Comment (ETsiunel) [reply hw_14/1.py:1]: it's not clear, why module name "1" is bad?

File: hw_14/2.py
Comment (ETsiunel) [reply hw_14/2.py:1]: ohh, i see((
I'll try to fix it

Comment (ETsiunel) [reply hw_14/2.py:1]: fixed

File: hw_14/1.py
Comment (Stml89) [reply hw_14/1.py:1]: if you were me or someone else, can you say what this module does? Why is it used without looking into the code?

**Review by Stml89 — CHANGES_REQUESTED** [review]
Pls, rename modules

**Review by Stml89 — APPROVED** [review]

**Pull request merged**